src/lib.rs

```diff
@@ -216,6 +221,7 @@ extern "system" {
     pub fn libusb_init(context: *mut *mut libusb_context) -> c_int;
     pub fn libusb_exit(context: *mut libusb_context);
     pub fn libusb_set_debug(context: *mut libusb_context, level: c_int);
+    pub fn libusb_set_cb(context: *mut libusb_context, cb: libusb_log_cb, mode: c_int);
```

Comment:
I was trying to use this function in Rust, and hit my head for a few hours before realizing it isn't in libusb our the resulting `.a` file anywhere? Is there a way to get it to use a callback logger?

Reply:
I think this function must be name `libusb_set_log_cb`

Reply:
Yes, I think you're correct. If I get it to work, I'll file a PR.

Reply:
Oh!
Sorry for this, my bad.
(I am wondering now how the hell I managed to *change* the name of this function during the  patch.)